keylime/db/verifier_db.py

```diff
@@ -57,5 +57,7 @@ class VerifierAllowlist(Base):
     id = Column(Integer, primary_key=True)
     agent = relationship("VerfierMain", back_populates="ima_policy")
     name = Column(String(255), nullable=False)
+    generator = Column(String(256))
+    checksum = Column(String(128))
```

Comment:
can we rename it to document the hash used? (I guess that it is sha256)

Reply:
not sure if we want to embed the choice of checksum on the column name

Reply:
I'm fine not having the hash type used in the name if we think it might change. But if it changes can we really support both? Queries against the data will have to use the right algorithm beforehand... but I guess it can send multiple values in the query (say sha256 or sha512).